lib/cc/analyzer/engine.rb

```diff
@@ -34,8 +34,12 @@ def run(stdout_io, container_listener)
         )
 
         container.on_output("\0") do |output|
-          unless output_filter.filter?(output)
-            stdout_io.write(output) || container.stop
+          next if output.blank?
+
+          issue = Issue.new(output)
+
+          unless output_filter.filter?(issue)
```

Comment:
You're right. I think it's `issue` that's actually the wrong name here, as mentioned in other comments.